plugins/jenkins/api-report.md

```diff
@@ -15,9 +15,29 @@ import { InfoCardVariants } from '@backstage/core-components';
 import { JSX as JSX_2 } from 'react';
 import { default as React_2 } from 'react';
 import { RouteRef } from '@backstage/core-plugin-api';
+import { TableColumn } from '@backstage/core-components';
 
+// Warning: (ae-missing-release-tag) "columnFactories" is part of the package's API, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
 // @public (undocumented)
-export const EntityJenkinsContent: () => JSX_2.Element;
+export const ciTableColumnFactories: Readonly<{
+  createTimestampColumn(): TableColumn<Project>;
+  createBuildColumn(): TableColumn<Project>;
+  createSourceColumn(): TableColumn<Project>;
+  createStatusColumn(): TableColumn<Project>;
+  createTestColumn(): TableColumn<Project>;
+  createActionsColumn(): TableColumn<Project>;
+}>;
+
+// Warning: (ae-missing-release-tag) "defaultCITableColumns" is part of the package's API, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
+export const defaultCITableColumns: TableColumn<Project>[];
```

Comment:
Don't export these, assign to the table component instead, like this: https://github.com/backstage/backstage/blob/49e92a6f85066e7ddd653a62783effce420b082b/plugins/github-deployments/src/components/GithubDeploymentsTable/GithubDeploymentsTable.tsx#L68